tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+from tempfile import NamedTemporaryFile
+
+from ocs_ci.framework.testlib import skipif_ocp_version, skipif_ocs_version, E2ETest
+from ocs_ci.framework.pytest_customization.marks import system_test
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.exceptions import CommandFailed
+
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@system_test
+@skipif_ocp_version("<4.13")
+@skipif_ocs_version("<4.13")
```

Comment:
We should have a teardown fixture, that would delete any resources created during the test, in case the test fails in the middle.

Reply:
Done.